pkg/services/alerting/conditions/reducer.go

```diff
@@ -62,6 +62,13 @@ func (s *SimpleReducer) Reduce(series *tsdb.TimeSeries) null.Float {
 	case "count":
 		value = float64(len(series.Points))
 		allNull = false
+	case "last":
+		for _, point := range series.Points {
```

Comment:
Perhaps we should loop backwards in this case? Not a big deal but it should be able to save a few cpu cycles :)